sw/host/opentitanlib/src/dif/lc_ctrl.rs

```diff
@@ -83,8 +88,296 @@ impl DifLcCtrlState {
             _ => Ok(DifLcCtrlState::StateInvalid),
         }
     }
+
+    /// Checks whether a transition is valid and whether a non-zero token is
+    /// required.
+    pub fn check_transition(self, target: DifLcCtrlState) -> DifLcCtrlTransCheck {
+        match target {
+            // TestUnlocked states
+            DifLcCtrlState::TestUnlocked0 => match self {
```

Comment:
These double matches should be matching a tuple:
```rust
match (self, target) {
    (DifLcCtrlState::Raw, DifLcCtrlState::TestUnlocked0) => ...,
    (_, DifLcCtrlState::TestUnlocked0) => ...,
}
```

Reply:
done